bundles/CoreBundle/Migrations/Version20220419120333.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use Pimcore\Model\DataObject\ClassDefinition\Listing;
+
+final class Version20220419120333 extends AbstractMigration
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function getDescription(): string
+    {
+        return 'Add index to versions.stackTrace to accelerate maintenance task "VersionsCleanupStackTraceDbTask"';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $versionsTable = $schema->getTable('versions');
+
+        if (!$versionsTable->hasIndex('stackTrace')) {
+            $versionsTable->addIndex(['stackTrace'], 'stackTrace', [], ['lengths' => [1]]);
```

Comment:
We need to add the index in the `install.sql` as well, see here: 
https://github.com/pimcore/pimcore/blob/265d95f285689b5df5c4f5b22c5e72c281bffb24/bundles/InstallBundle/Resources/install.sql#L757

Reply:
Done